java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -50,6 +53,67 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_sequence(JNIEnv *env, j
   CATCH_STD(env, 0);
 }
 
+JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_fromArrow(JNIEnv *env, jclass,
+                                                                        jint j_type,
```

Comment:
Nit: Indentation is off for these parameters

Reply:
somehow my vi kept adding in tabs instead of spaces at times, need to figure it out. Tried to replace with spaces but evidently messed it up. I'll fix.